README.md

```diff
@@ -75,6 +75,12 @@ Check out the [contribution guidelines](https://github.com/seisbench/seisbench/b
 
 ## Known issues
 
+- Some institutions and internet providers are blocking access to our data and model repository, as it is running on a non-standard port (2880).
+  This usually manifests in timeouts when trying to download data or model weights.
+  To verify the issue, try accessing [https://hifis-storage.desy.de:2880/](https://hifis-storage.desy.de:2880/) directly from the same machine.
+  As a mitigation, you can use our backup repository. Just run `seisbench.use_backup_repository()`.
+  Please note that the backup repository will usually show lower download speeds.
+  We recommend contacting your network administrator to allow access to port 2880 on our server as a more performant solution.
```

Comment:
Suggestion: "allow outgoing access" to port 2880.

"more performant" -> "higher performance".

Reply:
Also, "allow outgoing access to TCP port 2880". (Your network administrator will be happier opening a smaller hole. :)